source/glossary.rst

```diff
@@ -465,15 +462,14 @@ Glossary of Terms
      An Application Program Interface (API) is a specification and set of rules
      for how software programs interact with each other. In Mixxx, an example of
      this is the Sound API that Mixxx uses to communicate with the
-     :term:`operating system` to make use of the :term:`soundcards` connected to
+     :term:`operating system` to make use of the :term:`soundcards <soundcard>` connected to
      the computer.
 
    macOS
      macOS (originally named "Mac OS X" until 2012 and then "OS X" until 2016)
      is the current Mac operating system that officially succeeded the classic
      Mac OS in 2001.
 
-   EQ
```

Comment:
I think we should keep "EQ" *and* "equalizer" because one is an abbreviation and the other one is the full word.

Reply:
How about something like this? Equalizer (EQ) because if you ask me as a new user, i think the image below is an eyesore. Or while we leave EQ seperately from Equalizer, we can give an explanation like the one below.

EQ
This is the abbreviation for the :term:`equalizer`
![mixxx1](https://user-images.githubusercontent.com/62933980/96972695-baa47e80-150e-11eb-9114-16a5f6d346e8.PNG)

Your suggestions are welcome @Holzhaus, please review